secret-squirrel.cjs

```diff
@@ -1,8 +1,8 @@
 module.exports = {
 	files: {
 		allow: [
-			'.snyk',
-			'CODEOWNERS'
+			'.husky/pre-commit',
```

Comment:
we need to add old allowed files .snyk and CODEOWNERS

Reply:
why?

Reply:
is not needed , isn't any issue that squirrel could raise